AssetMappr/presentation/layout.py

```diff
@@ -35,22 +35,25 @@
 # =============================================================================
 # Function
 # =============================================================================
+
+
 def makeLayout(df, master_categories):
 
     return html.Div([
-        
+
         # Layout is split into two overall tabs
         dcc.Tabs([
 
             # Tab 1: Page to view, rate, and upload assets
-            dcc.Tab(label='Tell us about your community', children=[
-                
+            dcc.Tab(id='tab1', label='Tell us about your community', children=[
+
```

Comment:
Haha sorry but maybe the tab text has become too big now! So the layout is looking weird. Maybe we can just change it back to the original size for now, and see what Rick says?

Reply:
Yeahhhh. tricky thing is that I used the monitor to see our web, and now I find everything is soo big when I use my laptop to see our webpage. I am adjusting them now!